backend/controller/controller.go

```diff
@@ -1810,54 +1811,19 @@ func (s *Service) syncRoutes(ctx context.Context) (ret time.Duration, err error)
 				}
 			}
 			newRoutes[v.Module] = Route{Module: v.Module, Deployment: v.Key, Endpoint: targetEndpoint}
+			modulesByName[v.Module] = v.Schema
 		}
 	}
-	s.routes.Store(newRoutes)
-	return time.Second, nil
-}
-
-// Synchronises Service.schema from the database.
-func (s *Service) syncSchema(ctx context.Context) {
-	logger := log.FromContext(ctx)
-	modulesByName := map[string]*schema.Module{}
-	retry := backoff.Backoff{Max: time.Second * 5}
-	for {
-		err := s.watchModuleChanges(ctx, func(response *ftlv1.PullSchemaResponse) error {
```

Comment:
Is this being used anymore? If not we should remove it.

Reply:
It's used for PullSchema